### PR TITLE
Load data home page

### PR DESCRIPTION
**Pull request by HienDao97**
Load data for homepage
Fix DB
Screenshot from 2019-07-29 09-31-44

### CODE REVIEW

**Comment by HienDao97** [comment]
ready

File: app/models/post.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@ class Post < ApplicationRecord
   belongs_to :user
   belongs_to :travel_place
 
+  scope :popular_posts, ->{order(rate: :desc).take(6)}
```
Comment (hungct-1121) [inline app/models/post.rb]: 6 đưa vào config

File: app/models/travel_place.rb [hunk 2]
```diff
@@ -3,4 +3,6 @@ class TravelPlace < ApplicationRecord
   belongs_to :city
 
   has_many :travel_place_images, dependent: :destroy
+
+  scope :popular_places, ->{order(rate: :desc).take(6)}
```
Comment (hungct-1121) [inline app/models/travel_place.rb]: tương tự

**Comment by HienDao97** [comment]
ready

**Pull request merged**